src/Mod/Draft/WorkingPlane.py

```diff
@@ -241,6 +241,40 @@ def align_to_edge_or_wire(self, shape, offset=0):
         self.position = pos + (self.axis * offset)
         return True
 
+    def align_to_face_and_edge(self, face, edge, offset=0):
+        """Align the WP to a face and an edge.
+
+        The edge must be a border of the face, and the face must be planar.
```

Comment:
The function will also work if the edge is not a border of the face. This flexibility is actually useful, see one of the next comments.